PULL_REQUEST_MANAGEMENT.md

```diff
@@ -14,3 +14,8 @@ In addition, in order to avoid coming off as rude to helpful contributors, pleas
 - Asking the contributor to do unrelated work
 - Closing without providing a reason
 - Merging with the intention to rewrite that code soon after
+
+Before merging a PR, make sure it follows the [contributing rules](CONTRIBUTING.md#rules-and-recommendations), but more importantly:
+- Make sure the author is not using a nickname or alias, but a full legal name instead, and have specified a real e-mail in all commits
```

Comment:
I would vote to weaken that sentence on line19 to:
Make sure the author is using a real e-mail in all commits

I think real names instead of aliases should be preferred yet, but I think it is too strict to forbid aliases entirely.

Reply:
From the legal point, it would be better to forbid aliases, however some projects do allow aliases in case the author has sent a letter/document to the project's legal team, asking to associate an alias with the author.